.env.example

```diff
@@ -0,0 +1,24 @@
+TOKEN = 
+SENTRY = 
```

Comment:
see here we put that sentry link , so in the bot file replace that link with `sentry_link` variable since it will be cominbg from here :)